packages/algob/src/lib/algo-operator.ts

```diff
@@ -19,14 +25,18 @@ export function createAlgoOperator (network: Network): AlgoOperator {
 export interface AlgoOperator {
   algodClient: algosdk.Algodv2
   deployASA: (name: string, asaDesc: ASADef, flags: ASADeploymentFlags, account: Account) => Promise<ASAInfo>
+  deployASC: (programb64: string, scParams: object, flags: ASCDeploymentFlags, payFlags: ASCPaymentFlags,
+    account: Account) => Promise<ASCInfo>
   waitForConfirmation: (txId: string) => Promise<algosdk.ConfirmedTxInfo>
+  ensuredCompiled: (name: string, force: boolean) => Promise<ASCCache>
```

Comment:
This is not used anywhere except your own code when you can use private method and not have it in interface.

Reply:
This is also used in AlgoOperatorDryRunImpl to return mock file while testing

Reply:
But it's not useful outside of this. It's not useful code. Nothing new happens when it exists.

Reply:
But then how will I pass the mock files and test the code , the initial problem (the hardcoded network) will occur.

Reply:
Done